ansible_collections/arista/avd/roles/eos_cli_config_gen/README_v4.0.md

```diff
@@ -3050,6 +3050,11 @@ router_isis:
   address_family: < List of Address Families >
   isis_af_defaults:
     - maximum-paths < Integer 1-128 >
+  redistribute_routes:
+    - source_protocol: < route_type >
+      route_map: < route_map_name >
+      include_leaked: < true | false >
+      ospf_route_type: < internal | external | nssa-external >
```

Comment:
As we are modifying both READMEs lets make them the same
```suggestion 
      # ospf_route_type is required with source_protocols 'ospf' and 'ospfv3'
      ospf_route_type: < internal | external | nssa-external >
```

Reply:
Sync'ed up both README.md and READ_v4.0.md!